rpt/PharmaceuticalStockWard.properties

```diff
@@ -4,15 +4,11 @@ AvgValue = AVG VALUE
 
 Date = Date
 
-FootNotes1 = Value estimation using the unit costs average transferred to this \
-             ward for the given product
+FootNotes1 = Value estimation using the unit costs average transferred to this ward for the given product
 
-FootNotes2 = Value estimation using the unit costs weighted average \
-             transferred to this ward for the given product (bigger quantities \
-             contributes more)
+FootNotes2 = Value estimation using the unit costs weighted average transferred to this ward for the given product (bigger quantities contributes more)
```

Comment:
`(bigger quantities contributes more)` should be `(bigger quantities contribute more)`.   That is `contribute` should be singular.

Reply:
Thanks!